react/week3/Header.js

```diff
@@ -0,0 +1,6 @@
+export default function Header(props) {
```

Comment:
I am not sure that you need the Header component at all. You have only one h1 with the title that will probably not be changed.